qa/src/test/resources/credentials.properties

```diff
@@ -0,0 +1,22 @@
+## ---------------------------------------------------------------------------
```

Comment:
I know ... personally I don't think that a file like this should have a copyright header ... but you can do this .. if the first comment is written in your own words, then that should be fine.